src/main/java/com/openlattice/shuttle/dates/JavaDateTimeHelper.java

```diff
@@ -74,10 +83,11 @@ public OffsetDateTime parseDateAsDateTime( String date ) {
 
                 return ldt.atZone( tz.toZoneId() ).toOffsetDateTime();
             } catch ( Exception e ) {
-                logger.debug( "Unable to parse date as datetime {} with format string {}", date, datePatterns[ i ], e );
+                if (i == datePatterns.length - 1){
+                  logger.error( "Unable to parse date {}, please see debug log for additional information.", date );
```

Comment:
Can you change this error to say "Unable to parse date as datetime" -- so we know specifically which parsing method is failing